actors/builtin/multisig/multisig_actor_state.go

```diff
@@ -21,11 +22,11 @@ type MultiSigActorState struct {
 
 func (st *MultiSigActorState) AmountLocked(elapsedEpoch abi.ChainEpoch) abi.TokenAmount {
 	if elapsedEpoch >= st.UnlockDuration {
-		return abi.TokenAmount(0)
+		return abi.NewTokenAmount(0)
 	}
 
 	lockedProportion := (st.UnlockDuration - elapsedEpoch) / st.UnlockDuration
```

Comment:
Flagging that I think this integer division is wrong.

Reply:
Filed issue for this and will leave a comment pointing to it https://github.com/filecoin-project/specs-actors/issues/32